lstchain/scripts/lstchain_find_pedestals.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""
+This script has to be launched (with no arguments) on a directory containing
```

Comment:
I think it would be better to create a tool instead of a script

Reply:
What would be the advantage of making it a tool? It will not be part of the pipeline, it just produces input files for it, that will be fixed forever for any given run.